### PR TITLE
chore(cdk): `schematics` add `Avatar` migration

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [x] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

## What is the current behaviour?

part of #5808

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/cdk/schematics/ng-update/v4/tests/schematic-migrate-avatar.spec.ts [hunk 1]
```diff
@@ -0,0 +1,140 @@
+/* eslint-disable rxjs/no-topromise */
```
Comment (nsbarsukov) [inline projects/cdk/schematics/ng-update/v4/tests/schematic-migrate-avatar.spec.ts:1]: I think that we should disable this rule for all `**/schematics/**/*.spec.ts` files.

Now every test file for schematics contains this line.

**Review by nsbarsukov — APPROVED** [review]

File: projects/cdk/schematics/ng-update/v4/tests/schematic-migrate-avatar.spec.ts (hunk 1 above)
Comment (nsbarsukov) [reply projects/cdk/schematics/ng-update/v4/tests/schematic-migrate-avatar.spec.ts:1]: @splincode what do you think ?

Could we update this file ?
https://github.com/taiga-family/linters/blob/main/projects/eslint-plugin-experience/configs/angular/rxjs/index.js

Comment (nsbarsukov) [reply projects/cdk/schematics/ng-update/v4/tests/schematic-migrate-avatar.spec.ts:1]: Every test file for schematics contains:

```ts
const tree = await runner
    .runSchematicAsync(...)
    .toPromise();
```

Comment (splincode) [reply projects/cdk/schematics/ng-update/v4/tests/schematic-migrate-avatar.spec.ts:1]: I will do it right now

**Pull request merged**